From 3c9656769d33db55f6a9390a0f6d2c0be14d85c8 Mon Sep 17 00:00:00 2001
From: Thierry Vignaud <thierry.vignaud@gmail.com>
Date: Fri, 30 Nov 2018 15:41:05 +0100
Subject: [PATCH] kill bogus comment

should have been killed in commit 4a855e6bebed956c108865823bcc131ebea2841a

(cherry picked from commit a6bc200c28ac841edd12211c96dad8516095a68d)

Resolves: #1959336
---
 modules.d/05busybox/module-setup.sh | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/modules.d/05busybox/module-setup.sh b/modules.d/05busybox/module-setup.sh
index 14d401c2..ecbd6a13 100755
--- a/modules.d/05busybox/module-setup.sh
+++ b/modules.d/05busybox/module-setup.sh
@@ -22,9 +22,6 @@ install() {
         _progs="$_progs $_i"
     done
 
-    # FIXME: switch_root should be in the above list, but busybox version hangs
-    # (using busybox-1.15.1-7.fc14.i686 at the time of writing)
-
     for _i in $_progs; do
         _path=$(find_binary "$_i")
         [ -z "$_path" ] && continue

